src/styles/blocks/testimonials.scss

```diff
@@ -0,0 +1,66 @@
+.testimonials {
+  &__item {
+    margin-bottom: 48px;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+
+    @include onDesktop {
+      margin-bottom: 0;
+    }
+  }
+}
+
+.testimonial {
```

Comment:
there should be a separate file for each BEM-block

Reply:
You're right, thank's, done.